### ISSUE TITLE
Use cases

### ISSUE THREAD

**Issue by sknebel** [mbrehm/meet-here#13]
I thought about the project, how we described it and what it might be useful for a while and came up with 3 usage scenarios that are somewhat related, but require different things in different order. IMHO, the last one is "boring" from the usage perspective, but might make for a good first goal to get the basic technical stuff working.
## 1 "Doodle for places"

was the tagline we used at the #Mapathon
Now doodles are less spontaneous and more "planned"/voting, and a system offering that I would expect the following workflow:
1. someone creates a board (maybe selects a few points on the map as initial suggestions)
2. he/she distributes the link to others
3. these can see the options, possibly check routes to them (locally) and vote on them. Maybe add suggestions of their own or leave comments
4. after a while the creator checks back and informs everyone of the decision

Step 4 is optional, this also could happen quickly with everybody dropping suggestions on the board etc, thanks to websocket stuff
This _doesn't_ need: the isochrone 
## 2 The prototype

What we build at the Mapathon: really meant for real-time usage: Here I am, there is everyone else, what's good to reach for everybody. 
hard parts: the isochrone stuff, figuring out what's in that area and interesting to the users -> strong dependencies on potentially complex APIs
## 3 Position sharing

I know, there are already services for that. But it might be a good early prototype: Map "rooms" where everybody just can update their position and have a basic chat, updating for everybody.

**Comment by magmabyte** [comment]
I don't think we should limit ourselves to mimicking Doodle's functionality. As you don't get to choose a whole lot of options, our service is already more spontaneous than anything from Doodle. However, I think the Position sharing prototype is a good starting point to get familiar with the technologies on Frontend and Backend side and we can still tighten the problem that we want to address with the service and its idea.
